### DISCUSSION TITLE
My Visual Studio Language in Chinese

### DISCUSSION THREAD

**Discussion by bugalwena** [community/community#52149] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Most issues I can correct through research or others. I got a pop-up message to switch languages. I was thinking 'of software language, but it was the Chinese language. How do I revert the issue? It's not easy, as most titles and labels are in Chinese. I'm using memory to save files, but only a little can be done with half of the visual studio in a foreign language. I did a Google search, and it explains how configurations control what we build. Two of my extensions in my current project are Chinese, and I believe it may be the triggered the switch. My work is in an experimental phase; I need help. The answer could have been more helpful. I know what to do following the instructions. Could you give me visual directions? My studio editor labels such as "tools."' and commands are in Chinese, including my name! (please, no offence to any culture)

**Comment by HypnosCiceroQF** [comment]
I learned some Chinese,How can I help you?

**Reply by Yooo-Its-King** [reply]
I don't think they need help with understanding Mandarin/ Cantonese. I think they are trying to change their settings so that the UI is not in Mandarin/ Cantonese anymore.

**Reply by HypnosCiceroQF** [reply]
So I tried to tell him how to change the configuration.

**Comment by bugalwena** [comment]
Install language pack
